dtale/cli/loaders/excel_loader.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
```

Comment:
Nice catch! I switched to openpyxl because xlrd couldn't open xlsx files. I never had thought openpyxl doesn't support old xls file. I didn't have any xls files at hand to test so I assumed it would be backward compatible.